ee/tabby-webserver/src/repositories/resolve.rs

```diff
@@ -8,16 +14,41 @@ use axum::{
     Json,
 };
 use hyper::Body;
-use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
-use tabby_common::{config::Config, SourceFile, Tag};
+use tabby_common::{
+    config::{Config, RepositoryConfig},
+    SourceFile, Tag,
+};
+use tokio_cron_scheduler::{Job, JobScheduler};
 use tower::ServiceExt;
 use tower_http::services::ServeDir;
 
 use crate::repositories::ResolveState;
 
-lazy_static! {
-    static ref META: HashMap<RepositoryKey, RepositoryMeta> = load_meta();
+#[derive(Debug)]
+pub struct RepositoryCache {
+    repositories: RwLock<HashMap<RepositoryKey, RepositoryMeta>>,
+    pub configured_repositories: Vec<RepositoryConfig>,
```

Comment:
instead of making this public, extract a method for behaviors accessing this member